index.html

```diff
@@ -101,11 +101,17 @@ <h2 id="community-collaboration-guide">Community Collaboration Guide</h2>
 
   <p>A Red Hat guide to assist enterprise projects in collaborating with upstream communities on documentation.</p>
 
+  <h2 id="supplementary-style-guide-for-product-documentation">Red Hat supplementary style guide for product documentation</h2>
+
+  <p><a href="https://redhat-documentation.github.io/supplementary-style-guide/#introduction">Red Hat supplementary style guide for product documentation</a></p>
+
+  <p>This guide provides style guidelines and preferred term usage for Red Hat product and cross-product solution documentation. It is intended to be used as a supplement to the IBM Style Guide, which is the primary source of style guidance for Red Hat product documentation.</p>
+
   <h2 id="glossary-of-terms-and-conventions-for-product-documentation">Glossary of Terms and Conventions for Product Documentation</h2>
```

Comment:
Since you added the SSG above, I'd suggest that we remove this section completely. This makes it look like it's a separate guide, when it isn't.

Reply:
@bergerhoffer 
I removed the heading, but I still think it's worthwhile to call out the Glossary because:
- It's major
- People might expect to see it on this page, and we want to help them find it.

Reply:
@stoobie sure that's fine to keep it. But then can you rename it to how it actually looks in the guide: "Glossary of terms and conventions". And maybe consider bumping up the sentence below it to condense it a bit? Something like:

```
 <p>The style guide also includes the <a href="https://redhat-documentation.github.io/supplementary-style-guide/#glossary-terms-conventions">Glossary of terms and conventions</a>, which outlines terms and conventions for technical language in Red Hat's product documentation and other technical content. These terms and conventions supplement those in The IBM Style Guide, which is the main reference for technical terms.</p>
```
Maybe also consider updating the first paragraph to remove "and preferred term usage" to allow the first para to be about the style guidance, and the second para to be about the glossary?

Reply:
@bergerhoffer 

> @stoobie sure that's fine to keep it. But then can you rename it to how it actually looks in the guide: "Glossary of terms and conventions". And maybe consider bumping up the sentence below it to condense it a bit? Something like:
> 
> ```
>  <p>The style guide also includes the <a href="https://redhat-documentation.github.io/supplementary-style-guide/#glossary-terms-conventions">Glossary of terms and conventions</a>, which outlines terms and conventions for technical language in Red Hat's product documentation and other technical content. These terms and conventions supplement those in The IBM Style Guide, which is the main reference for technical terms.</p>
> ```

Fixed. 

> Maybe also consider updating the first paragraph to remove "and preferred term usage" to allow the first para to be about the style guidance, and the second para to be about the glossary?

OK, but I copied that first paragraph from the SSG. Just FYI.

Reply:
Yeah I get that. But you're also adding a second para specific to just the terms, which the SSG doesn't do. So if it's important enough to call them out separately, then I'd suggest to separate the style guidelines in the first para, and the terms in the second.